prac_05/word_occurences.py

```diff
@@ -0,0 +1,29 @@
+"""Word occurrences: Enter a sentence and see how many times your word is in that sentence."""
+
+
+def main():
+    sentence = input("Text: ")
+    words_occurrences_dict = count_occurrences(sentence)
+    display_dictionary(words_occurrences_dict)
+
+
+def count_occurrences(sentence):
+    """Count how often a word occurs in the given sentence."""
+    words = sentence.split(" ")
```

Comment:
" " in .split(" ") is unnecessary as the function automatically uses a space to split if no argument is specified